Description: Fix FTBFS with gcc 4.7 due to missing <unistd.h> include
Author: Cyril Brulebois <kibi@debian.org>
Last-Update: 2012-04-15

--- a/src/RemoteServer.cpp
+++ b/src/RemoteServer.cpp
@@ -29,6 +29,7 @@
 #include <stdio.h>
 #include <errno.h>
 #include <limits.h>
+#include <unistd.h>
 
 #ifdef _MSC_VER
 #include <io.h>  // for close()
